db_pre_commit.bat

```diff
@@ -0,0 +1,4 @@
+@echo off
+cls
+rake db_pre_commit
```

Comment:
Entry point just for you. The only think that a bat file should do in PKSim, **for consistency and maintainability**,  is call rake tasks